src/main/groovy/com/stormpath/tck/util/ApiKey.groovy

```diff
@@ -60,13 +60,21 @@ class ApiKey {
            apiKeySecret = properties.get(API_KEY_SECRET_PROPERTY_NAME) != null ? properties.get(API_KEY_SECRET_PROPERTY_NAME) : apiKeySecret
        }
 
+       //4. Try system properties:
+       properties = getSystemProperties();
+       if (properties.size() > 0) {
+           apiKeyID = properties.get(API_KEY_ID_PROPERTY_NAME) != null ? properties.get(API_KEY_ID_PROPERTY_NAME) : apiKeyID
```

Comment:
The Java SDK does support reading in ApiKey credentials via system properties, but this is separate. It is just a convenience mechanism to read in the creds the TCK uses to communicate with IAM and not part of the testing the integration, so not sure I quite understand the objection?